test/performance/benchmarks/scale-from-zero/dev.config

```diff
@@ -0,0 +1,36 @@
+# Creating this benchmark:
+# mako create_benchmark \
+#   test/performance/benchmarks/scale-from-zero/continuous/dev.config
+# Updating this benchmark:
+# mako update_benchmark \
+#   test/performance/benchmarks/scale-from-zero/continuous/dev.config
+project_name: "Knative"
+benchmark_name: "Development - Serving scale from zero"
+description: "Scale from zero test against ksvcs in parallel."
+benchmark_key: '5024954898710528'
+
+# Human owners for manual benchmark adjustments.
+owner_list: "mattmoor@google.com"
```

Comment:
Matt's email won't work anymore :(
So remove it.

Reply:
Removed :(